ci-operator/step-registry/openshift/microshift/infra/wait-for-cluster-up-aws/openshift-microshift-infra-wait-for-cluster-up-aws-commands.sh

```diff
@@ -45,6 +45,7 @@ else
   done
   sudo ls -la /var/lib/microshift
   sudo ls -la /var/lib/microshift/resources/kubeadmin/kubeconfig
+  sudo systemctl start greenboot-healthcheck
```

Comment:
Should we use "restart" here just in case greenboot was started before?

Reply:
Done